Components/Avatar/landingPage.css

```diff
@@ -0,0 +1,217 @@
+}
+
+
+body
+{
+margin: 0;
+padding: 0;
+    
+
+}
+
+:root
+{
+    --primary-color: #63DCCD;
+    --secondary-color: #266350;
+}
+
+.logo-component
+{
+
+    height: 50px;
+    width: 60px;
+}
+
+.navigation-menu
+{
+    padding: 1.5rem 1rem;
+    /* width: 100%; */
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    align-items: center;
+    /* border-bottom: 2px solid var(--primary-color); */
+}
+
+.left-side-nav
+{
+    display: flex;
+
+    justify-content: space-between;
+    align-items: center;
+}
+
+.right-side-nav
+{
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.sub-texts
+{
+    margin: 0.2rem;
+    color: var(--secondary-color);
+    text-decoration: none;
+    font-size: 1rem;
+    font-weight: 600;
+}
+
+
+.sub-texts:hover
+{
+    border-bottom: 2px solid var(--secondary-color);
+}
+
+.heading
+{
+    font-size: 2rem;
+    text-decoration: none;
+    font-weight: 700;
+    font-family: 'Lato', sans-serif;
+    font-style: italic;
+    color: var(--secondary-color);
+}
+
+.info-container
+{
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+   text-align: center;
+    padding-top: 7rem;
+   
+}
+
+.main-caption
+{
+    font-size: 3.5rem;
+    font-style: italic;
+    font-weight: 400;
+    /* line-height: 3.5rem; */
+    align-self: center;
+     width: 70%;
+color: var(--primary-color);
+}
+
+.sub-caption
+{
+    font-size: 1.35rem;
+    font-weight: 500;
+    padding: 1rem;
+    align-self: center;
+     width: 70%;
+
+}
+
+.btn
+{
+    font-size: 1.35rem;
+    padding: 0.7rem 1.4rem;
+    background-color: var(--primary-color);
+    border-radius: 7px;
+    border: var(--primary-color);
+
+
+}
+.btn-next:hover
+{
+    background-color: white;
+    border: 3px solid var(--primary-color);
+}
+
+.installation-container
+{
+    margin-top: 1rem;
+}
+
+
+
+
+.footer-container
+{
+    margin-top: 1rem;
+    padding: 2rem;
+    background-color:var(--primary-color);
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+}
+
+.footer-icons
+{
+    display: flex;
+    flex-flow: row;
+    gap: 10px;
+}
+
+.small-icons
+{
+    font-size: 1.5rem;
+    color: black;
+
+}
+
+@media screen and (max-width : 800px){
+
+    .navigation-menu{
+        justify-content: flex-start;
+        gap: 10px;
+        padding: auto;
+    }
+
+    .left-side-nav .heading{
+        font-size: 1rem;
+    }
+
+ 
+
+    .right-side-nav a{
+        font-size: 0.8rem;
+        margin: 0.4rem;
+    }
+
+    .info-container
+    {
+        width: auto;
+    }
+
+    .info-container .main-caption{
+        font-size: 2rem;
+    
+    }
+
+    .info-container .sub-caption{
+
+        font-size: 1rem;
+        
+    }
+.footer-icons a
+{
+font-size: 1rem;
+}
+
+.installation-container iframe
+{
+    width:50%;
+}
+
+}
+
+  
+
+@media screen and (max-width : 600px){
+ 
```

Comment:
Remove the unnecessary blank spaces in between

Reply:
Thank you will look into it